en_us/en_us.rc

```diff
@@ -1575,6 +1577,8 @@ BEGIN
         MENUITEM "Word Wrap\tCtrl+W",           ID_EDIT_WORDWRAP, CHECKED
         MENUITEM "Set Tab Size",                ID_ADVANCED_SETTABSIZE
         MENUITEM "Toggle Whitespace Chars",     ID_ADVANCED_SHOWWS
+        MENUITEM SEPARATOR
+        MENUITEM "Copy Path",                   ID_REPOSITORY_COPYPATH
```

Comment:
You need to get the ID_XXX onto the Ribbon somewhere for the accelerator to get registered...

Reply:
@GordonSmith gotcha.